app/src/main/res/layout/row_harvested_tweet.xml

```diff
@@ -15,42 +16,47 @@
 
         <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:weightSum="1">
 
             <TextView
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_weight="1"
                 android:id="@+id/user_fullname"
-                android:ellipsize="end" />
-
-            <TextView
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_weight="1"
-                android:id="@+id/username"
-                android:ellipsize="end" />
+                android:layout_weight=".5"
+                android:ellipsize="end"
+                android:textStyle="bold"
+                android:textColor="@android:color/black"
+                tool:text="User name"/>
 
             <TextView
-                android:layout_width="wrap_content"
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:id="@+id/tweet_date" />
+                android:layout_weight=".5"
+                android:gravity="end"
+                android:id="@+id/tweet_date"
+                android:layout_marginStart="@dimen/location_container_margin"
+                tool:text="26 July, 2017"/>
 
         </LinearLayout>
 
-        <LinearLayout
+        <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="4dp">
+            android:id="@+id/username"
+            android:ellipsize="end"
+            android:layout_marginBottom="@dimen/location_container_margin"
+            tool:text="'@User_name'"/>
```

Comment:
Why making it `@User_name` while it actually is `@user_name`?

Reply:
@singhpratyush  In tool:text we cannot use "@" directly as to show as it is text in the textView so to just have a look while making UI I used '@User_name' but in app it will come @user_name only.